WTWSMS/common/minor_titles/WtWSMS_minor_titles.txt

```diff
@@ -132,32 +132,37 @@ title_sigecempa = {
 }
 
 title_court_celtic_poet = { # Celtic Poet/Bard
-	dignity = 0.25
+	dignity = 1
 	realm_in_name = yes
 	grant_limit = 1
 	opinion_effect = 30
 	
 	monthly_salary = 0.02
-	monthly_prestige = 0.01
+	monthly_prestige = 0.1
 	
 	allowed_to_hold = {
 		is_adult = yes
 		is_ruler = no
 		is_female = no
-		trait = poet
+		OR = {
+			trait = poet
+			learning = 14
+		}
```

Comment:
Anyone of sufficient literacy and imagination can make poems. In the case of this game, fourteen IQ is very high and demonstrates an interest in intellectual pursuits. This should stay so that the title isn't eternally unused.

However, I am open to an Event where the Court Poet, with certain Traits, doesn't want to make poetry and submits his resignation for your request. That would remove his Poet Trait as well.